From 186fd6ebaf10e05832dbcc7887a58012cfb12325 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Mon, 17 Oct 2011 14:40:27 +0200
Subject: [PATCH] theme.edc: fix edje_cc build

| /OE/shr-core/tmp/sysroots/x86_64-linux/usr/bin/edje_cc -v theme.edc gweled.edj
| ERR<6730>:edje_cc edje_cc_parse.c:157 new_object() /OE/shr-core/tmp/sysroots/x86_64-linux/usr/bin/edje_cc: Error. theme.edc:87 unhandled keyword | PARSE STACK:
| collections.images.| PARAMS:

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 data/themes/gweled/theme.edc |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/themes/gweled/theme.edc b/data/themes/gweled/theme.edc
index f60363d..70aa60b 100644
--- a/data/themes/gweled/theme.edc
+++ b/data/themes/gweled/theme.edc
@@ -84,9 +84,9 @@
 
 
 collections {
-    images { \
-        image: "border.png" COMP; \
-    } \
+    images {
+        image: "border.png" COMP;
+    }
 
    group { name: "board/bg";
 
-- 
1.7.7

